services/repository/adopt.go

```diff
@@ -36,10 +36,6 @@ func AdoptRepository(ctx context.Context, doer, u *user_model.User, opts CreateR
 		}
 	}
 
-	if len(opts.DefaultBranch) == 0 {
```

Comment:
Why remove these lines?

Reply:
This will be done here
![image](https://github.com/go-gitea/gitea/assets/18380374/3442fc79-0fa2-4fa1-978b-28a36ca538c9)